src/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -40,15 +47,16 @@ interface SavedAddressesProps {
   commitMutation?: CommitMutation
   relay: RelayRefetchProp
   addressCount?: number
-  onAddressDelete?: (removedAddressId: string) => void
-  onAddressCreate?: (
-    address: CreateUserAddressMutation$data["createUserAddress"]
-  ) => void
-  onAddressEdit?: (
-    address: UpdateUserAddressMutation$data["updateUserAddress"]
-  ) => void
+  onAddressDelete?: (id: string) => void
+  onAddressEdit?: (address: {
+    internalID: string
+    attributes: ShippingAddress
+  }) => void
+  onAddressCreate?: (address: {
+    internalID: string
+    attributes: ShippingAddress
```

Comment:
since the param types are same and used in the parent as well, maybe it's a good idea to have a type for this.

Reply:
good idea!